server/komodo-rest/src/main/java/org/komodo/rest/KomodoRestV1Application.java

```diff
@@ -177,11 +197,6 @@
         String DATA_SERVICE_SEGMENT = "dataservice"; //$NON-NLS-1$
 
         /**
-         * The name of the URI path segment for DataService clone in the Komodo workspace.
-         */
-        String CLONE_DATA_SERVICE_SEGMENT = "clone"; //$NON-NLS-1$
-
-        /**
```

Comment:
Can the CLONE... segments not be collapsed into 1. It may seem daft given they both have the same value but using different names describes which service they are used in.